frontend/ResourcesPage.css

```
@@ -360,4 +360,622 @@
   text-align: center;
   color: #888;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
```

Stylelint (17.10.0)

[error] Expected no quotes around "Poppins" (font-family-name-quotes)

(font-family-name-quotes)